### PR TITLE
Feat/Pending Info: hide tooltip in case that withdrawable amount is same as fiat balance

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Withdraw/EnterAmount/template.success.tsx [hunk 1]
```diff
@@ -103,6 +104,14 @@ const Success: React.FC<InjectedFormProps<
 
   const showPendingTransactions = !isEmpty(props.locks)
 
+  const showInfoTooltip =
+    Number(props.withdrawableBalance) < Number(props.availableBalance)
+
+  // eslint-disable-next-line
+  console.log('propssss', props)
+  // eslint-disable-next-line
+  console.log('showInfoTooltip', showInfoTooltip)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Withdraw/EnterAmount/template.success.tsx:113]: remove these logging lines

**Review by schnogz — CHANGES_REQUESTED** [review]

**Review by Ogungodspower5 — APPROVED** [review]
Cashapp:$Trystan77

File: packages/blockchain-wallet-v4-frontend/src/modals/Withdraw/EnterAmount/template.success.tsx (hunk 1 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/modals/Withdraw/EnterAmount/template.success.tsx]: @milan-bc you shouldn't need //eslint-disable-next-line for running locally, so you can still console.log w/o adding eslint-disable.

When you try to commit however, commitlint should warn you about the logs, so you can remove them before comitting and pushing.

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Withdraw/EnterAmount/template.success.tsx]: @plondon cool, will check 👍

**Pull request merged**